schemas/groups/electrical.json

```diff
@@ -238,263 +242,427 @@
           "units": "W"
         }
       }
+    },
+  
+    "chargerQuantities": {
+      "type": "object",
+      "title": "Charger Quantities",
+      "description": "Common charger quantities",
+      "properties": {
+        "allOf": [{
+            "$ref": "#/definitions/identity"
+            }],
+        "chargingAlgorithm": {
+          "type": "object",
+          "description": "Algorithm being used by the charger",
+          "allOf": [
+            {
+              "$ref": "../definitions.json#/definitions/commonValueFields"
+            },
+            {
+              "properties": {
+                "value": {
+                  "type": "string",
+                  "enum": [
+                    "trickle",
+                    "two stage",
+                    "three stage",
+                    "four stage",       
+                    "constant current",
+                    "constant voltage",
+                    "custom profile"
+                  ]
+                }
+              }
+            }
+          ]
+        },
```

Comment:
@tkurki  'chargerMode' defines the relationship between multiple (and paired) charging sources.  An example is when two inverter/chargers are configured to allow for the US power standard of 120v & 240v. Or to parallel two or more inverter / charger to increase the wattage capability.  In that case there is often a small cable between the two inverter/chargers and they run in master/slave mode.   Magnum refers this to 'Stacking', and other vendors have the same capability as well.  In this configuration, one acts as the master and the other(s) become slaves.

We could perhaps call it 'chargerConfiguraiton' instead if that is clearer.

The mode defines what phase a given charging source is operating in during a battery charge cycle (typically:  Bulk, Acceptance, and then Float).  I think the extra 'Charging ' and space is a carry over from before there was the common attribute of `chargerQuantities`, and I think it would be appropriate to remove that prefix, just leaving the Bulk, Acceptance, etc..